src/LOR/models.py

```diff
@@ -10,7 +10,7 @@ class LOR(models.Model):
     due_date = models.DateField()
     writer_email = models.EmailField()
     company_name = models.CharField(max_length=100)
-    company_website = models.CharField(max_length=100)
+    company_website = models.URLField()
```

Comment:
The LOR model contains most of the items that we need to describe the LOR application. I think we would just need a role to identify a user. In the future we could also break down the LOR model into smaller models like account, role, and request with unique identifiers so that it would follow a more traditional db approach.